### ISSUE TITLE
=../2 String: principal functor should be . (dot)?, arguments scrambled

### ISSUE THREAD

**Issue by flexoron** [infradig/trealla#413]
```
?- "abc" =.. L, writeq(L), nl.
["abc",[[]|false]]
   L = ["abc",[[],=('L',_0)]]. % yet an interesting answer
```

**Comment by flexoron** [comment]
writing infecting variable L ... why is that?
```
?- "abc" =.. L, write_canonical(L), nl.
'.'('.'(a,'.'(b,'.'(c,[]))),'.'('.'([],'.'(a,'.'(b,'.'(c,[])))),[]))
   L = ["abc",[[]|"abc"]].

?- "abc" =.. L, write(L), nl.
["abc",[[]|false]]
   L = ["abc",[[],=('L',_0)]].
```

**Comment by infradig** [comment]
Univ doesn't handle strings at the moment. L is wrong, so doesn't matter
how you write it.

**Comment by flexoron** [comment]
ok
just surprised at some kind of feedback happens according to the motto: hey L, its me, write_canonical, change your outfit.
```
?- "abc" =.. L.
   L = ["abc",[[],=('L',_0)]].
```

**Comment by flexoron** [comment]
fixed with v1.20.10
